demos/node-classification/gcn/gcn-cora-node-classification-example.ipynb

```diff
@@ -4,14 +4,36 @@
    "cell_type": "markdown",
```

Comment:
<blockquote>...is a list of hidden feature sizes of each layer in the model...</blockquote><p> </p><p>The layer is commonly referred to as hidden not the individual units. Saying something like, "<code>layer_sizes</code>  denotes the number of hidden layers and the sizes of each layer. In this example, we use two GCN layers with 16 units each."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stellargraph/stellargraph/pull/1125/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>